Web UI/CognitiveSearch.UI/wwwroot/js/results.js

```diff
@@ -43,13 +43,13 @@ function AuthenticateResultsMap(results) {
             $('#maps-viewer').html(mapsContainerHTML);
 
             // default map coordinates
-            var coordinates = [-122.32, 47.60];
+            // var coordinates = [-122.32, 47.60];
```

Comment:
Are we not defaulting to a specific location on map? What is the current behaviour then?

Reply:
no, we're showing a global map view, rather than zooming on a particular area